Tangra.Model/Config/TangraConfig.cs

```diff
@@ -1153,7 +1163,10 @@ public Color Target4Color
 
 			public static Font LabelsFont = new Font(FontFamily.GenericMonospace, 9);
 
-			public void Load()
+            public SaveAsImageFormat SaveAsImageType;
```

Comment:
Looks like this field is unused, where `SaveAsImageFileType` is used instead. Do I read this correctly? If so then this line can be deleted.